compiler/src/dotty/tools/dotc/ast/DesugarEnums.scala

```diff
@@ -6,6 +6,7 @@ import core._
 import util.Positions._, Types._, Contexts._, Constants._, Names._, NameOps._, Flags._
 import SymDenotations._, Symbols._, StdNames._, Annotations._, Trees._
 import Decorators._
+import dotty.tools.dotc.reporting.diagnostic.messages.EnumCaseDefinitionInNonEnumOwner
```

Comment:
Can we shorten this to just `reporting.diagnostic.messages.EnumCaseDefinitionInNonEnumOwner`?